source/docs/hardware-reference/talonfx/improving-performance-with-current-limits.rst

```diff
@@ -71,7 +71,7 @@ For example, a user may have the following mechanisms and supply limits.
 - x1 Kraken(s) on elevator - 30A supply
 - x1 Kraken(s) on intake - 15A supply
 
-This would yield peak supply current of ~395A for a worst case scenario. This draw is extremely unlikely as peak supply is often extremely brief (for example, 60A on all 4 swerve drive motors will likely be for less than 5 seconds).
+This would yield peak supply current of ~365A for a worst case scenario. This draw is extremely unlikely as peak supply is often extremely brief (for example, 60A on all 4 swerve drive motors will likely be for less than 5 seconds) and all mechanisms won't be under peak load at the same time.
```

Comment:
Maybe also mention something like "A more common scenario is driving all 8 swerve motors at the same time for a peak supply current of 320 A"